src/tezos/TezosOperations.ts

```diff
@@ -14,16 +14,16 @@ import {Operation} from "./TezosTypes";
  * Output of operation signing.
  */
 export interface SignedOperationGroup {
-    bytes: Buffer,
-    signature: string
+    bytes: Buffer;
+    signature: string;
 }
 
 /**
  * Result of a successfully sent operation
  */
 export interface OperationResult {
-    results: TezosTypes.AlphaOperationsWithMetadata,
-    operationGroupID: String
+    results: TezosTypes.AlphaOperationsWithMetadata;
+    operationGroupID: string;
```

Comment:
In the past I have seen variable behaviour depending on whether one uses `string` or `String`. Are we sure no behaviour has changed here?

Reply:
https://stackoverflow.com/questions/14727044/typescript-difference-between-string-and-string